eng/Packages.Data.props

```diff
@@ -102,6 +102,8 @@
     <PackageReference Update="Azure.Storage.Queues" Version="12.14.0" />
     <PackageReference Update="Azure.ResourceManager" Version="1.5.0" />
 
+    <PackageReference Update="Moq" Version="4.18.2" />
```

Comment:
Moq already appears under the Test project section (L258).   We do not want to add it here, as it should not be referenced by our non-test projects.